pkg/plugins/docker/dockerfile/simpletextparser/main.go

```diff
@@ -135,6 +135,7 @@ var supportedKeywordsInitializers = map[string]keywords.Logic{
 	"user":        nil,
 	"workdir":     nil,
 	"arg":         keywords.Arg{},
+	"env":         keywords.Env{},
```

Comment:
This key already exists in the map, which leas to compilation error with the error message `duplicate key "env" in map literal`.